virt/virt-4-11-release-notes.adoc

```diff
@@ -105,6 +105,15 @@ Deprecated features are included in the current release and supported. However,
 Removed features are not supported in the current release.
 
 //CNV-16317 NMState is no longer part of CNV
+* {VirtProductName} {VirtVersion} removes support for link:https://nmstate.io/[nmstate] including the following objects:
++
+--
+** `NodeNetworkState`
+** `NodeNetworkConfigurationPolicy`
+** `NodeNetworkConfigurationEnactment`
+--
++
+To continue using nmstate features, you must first install the xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[Kubernetes NMState Operator] from the *OperatorHub* in the {product-title} web console, or by using the OpenShift CLI (`oc`), before updating to {VirtProductName} {VirtVersion}. The Kubernetes NMState Operator preserves and supports the existing nmstate configuration.
```

Comment:
Do I understand correctly that regardless of the method of (NMState Operator) installation, the NMState Operator must be installed _before_ the cluster is updated to 4.11? I think the first sentence is a bit ambiguous and might benefit from being broken into two sentences. 

Suggestion:
```
... you must install the ... [Kubernetes NMState Operator] before updating to {VirtProductName} {VirtVersion}. You can install it from the *OperatorHub* in the {product-title} web console or by using the OpenShift CLI (`oc`). The Kubernetes NMState Operator preserves ...
```